cmd/upgrade/cluster/cmd.go

```diff
@@ -280,17 +277,32 @@ func run(cmd *cobra.Command, _ []string) {
 					os.Exit(1)
 				}
 			}
-			reporter.Infof("Preparing to upgrade role policies")
 			accountroles.Cmd.Run(accountroles.Cmd, []string{prefix, mode})
 			if mode == aws.ModeManual {
 				reporter.Infof("Run the following command to continue scheduling cluster upgrade"+
 					" once account and operator roles have been upgraded : \n\n"+
 					"\trosa upgrade cluster --cluster %s\n", clusterKey)
 				os.Exit(0)
 			}
+		} else {
+			reporter.Infof("Account and/or operator roles are up-to-date")
```

Comment:
Getting to this `else` means that both account _and_ operator roles are up to date. You should probably change that "and/or" to just "and"